utils/githooks/pre-commit

```diff
@@ -7,12 +7,22 @@ export TARGET
 hook=${0##*/}
 rm -f ".${hook}"
 
+IFS=', ' read -r -a skip_list <<< "${DAOS_SKIP:-none}"
+echo "skip_list=${skip_list[*]}"
+
 run-parts() {
     local dir="$1"
     shift
 
     for i in $(LC_ALL=C; echo "${dir%/}"/*[^~,]); do
-        # don't run vim .swp files
+        skip_item=0
+        for skip in "${skip_list[@]}"; do
+            if [[ "${i}" =~ ${skip} ]]; then
+                skip_item=1
+                break
+            fi
```

Comment:
Should this print what is being skipping?

Reply:
not a bad idea